plugins/package-managers/composer/src/main/kotlin/Composer.kt

```diff
@@ -329,19 +329,22 @@ private fun parseVirtualPackageNames(
         removeAll(packages.keys)
     }
 
-private fun PackageInfo.toPackage(): Package {
+private fun PackageInfo.toId(): Identifier {
```

Comment:
The mentioned "following change" is not part of this PR. Is that intended? That makes it hard to judge whether the refactoring is reasonable.

Reply:
It was not my intention to make you remove that sentence from the commit message. Instead I propose to
- either move this commit to the PR that actually makes reuse of the extracted function, or
- provide a rationale why the extraction makes sense already now.